docker/caffe2/jenkins/centos-rocm/.gitignore

```diff
@@ -0,0 +1 @@
+*.sh
```

Comment:
this gitingore file doesn't seem needed

Reply:
It comes with the others. There is some soft-linking of shell scripts going on during the build, so the gitignore for them makes sense to me?

Reply:
oh you are right, I didn't notice other directories have this as well